suite-native/navigation/src/components/ScreenContent.tsx

```diff
@@ -51,7 +51,6 @@ export const ScreenContent = ({
                     keyboardShouldPersistTaps="always"
                     contentInsetAdjustmentBehavior="automatic"
                     contentContainerStyle={screenStyle}
-                    viewIsInsideTabBar
```

Comment:
Won't this break transaction list now?

Reply:
Also please check the whole app for bottom paddings. I think this is what @mnuky mentioned as bug of white space in settings and account import

Reply:
No this is related only to the scrollbar. TransactionList and all the other screens work fine.

<img width="200" alt="Screenshot 2023-03-22 at 12 17 38" src="https://user-images.githubusercontent.com/26143964/226902998-b6beb49c-ccb5-4aa8-afba-67f1ac3de88c.png">